devtools/dataset-publisher/app.py

```diff
@@ -25,7 +25,7 @@ async def dev_simulate_publish_dataset(request: Request, filename: str = None):
 
     # If filename is not provided, create a guid as the filename before we publish it
     if filename in (None, ""):
-        filename = f"{str(uuid.uuid4())}.json"
+        filename = f"{uuid.uuid4()!s}.json"
```

Comment:
what was it complaining about here ?

Reply:
https://docs.astral.sh/ruff/rules/explicit-f-string-type-conversion/ - use of str() function rather than conversion flag within f-string